go/protocol/keybase1/user.go

```diff
@@ -144,6 +144,7 @@ type ListTrackingJSONArg struct {
 type SearchArg struct {
 	SessionID int    `codec:"sessionID" json:"sessionID"`
 	Query     string `codec:"query" json:"query"`
+	Service   string `codec:"service" json:"service"`
```

Comment:
Ok now properly re-ran it. Will find another person to review once I confirm everything is still working. :)